### ISSUE TITLE
Add Bag-Of-Words functionality to playlist class

### ISSUE THREAD

**Issue by trackoverxc** [malctaylor15/Song_Lyrics_NLP#16]
TFIDF = Term frequency inverse document frequency.

**Comment by malctaylor15** [comment]
We can follow SkLearn's implementation here -- http://scikit-learn.org/stable/auto_examples/model_selection/grid_search_text_feature_extraction.html#sphx-glr-auto-examples-model-selection-grid-search-text-feature-extraction-py

**Comment by trackoverxc** [comment]
"ikit-learn provides utilities for the most common ways to extract numerical features from text content, namely:

    tokenizing strings and giving an integer id for each possible token, for instance by using white-spaces and punctuation as token separators.
    counting the occurrences of tokens in each document.
    normalizing and weighting with diminishing importance tokens that occur in the majority of samples / documents."

**Comment by trackoverxc** [comment]
"A corpus of documents can thus be represented by a matrix with one row per document and one column per token (e.g. word) occurring in the corpus."

**Comment by trackoverxc** [comment]
Our data is formatted incorrectly to do the tokenizing. We need 1 string per song (document) and all those strings stored as a list which is feed into the CountVectorizer. This may require rethinking the song and playlist classes to get the correct formatting in the beginning. @malctaylor15

**Comment by malctaylor15** [comment]
I think the structure can be used to get a string for each song. Each playlist (class) has an attribute called listOfSongs which is a list of song instances (for each song). From that list, we can extract the lyrics from each song and have a list of strings where each string is a song's lyrics. 

It could work something like this -- 
testPlaylist = playlist(spotify_tracklist_id, username)
songlist = testPlaylist.listOfSongs
song_lyrics_list = [song.lyrics for song in songlist]

**Comment by trackoverxc** [comment]
Added the word frequency functionality

**Comment by trackoverxc** [comment]
Finished w/ Malcolm
